src/app/ui/styles/modaldialog-styles.js

```diff
@@ -12,47 +10,37 @@ var css = csjs`
     width: 100%; /* Full width */
     height: 100%; /* Full height */
     overflow: auto; /* Enable scroll if needed */
-    background-color: ${styles.colors.black}; /* Fallback color */
-    background-color: ${styles.colors.opacityBlack}; /* Black w/ opacity */
     word-wrap: break-word;
   }
   .modalHeader {
     padding: 2px 16px;
-    background-color: ${styles.remix.modalDialog_Header_Footer_BackgroundColor};
-    color: ${styles.remix.modalDialog_Header_Footer_Color};
     display: flex;
     justify-content: space-between;
   }
   .modalBody {
-    background-color: ${styles.remix.modalDialog_BackgroundColor_Primary};
-    color: ${styles.remix.modalDialog_text_Primary};
     padding: 1.5em;
     line-height: 1.5em;
   }
-  .modalBody em{
-    color: ${styles.remix.modalDialog_text_Em};
+  .modalBody em {
+    color: var(--text-info)
   }
-  .modalBody a{
-    color: ${styles.remix.modalDialog_text_Link};
+  .modalBody a {
+    color: color: var(--text-info)
```

Comment:
this is the link color - I think it needs to be a different color that the normal text.

Reply:
`--test-warning` ?